client/src/main/java/org/apache/rocketmq/client/consumer/DefaultMQPushConsumer.java

```diff
@@ -461,7 +461,11 @@ public void start() throws MQClientException {
      */
     @Override
     public void shutdown() {
-        this.defaultMQPushConsumerImpl.shutdown();
+        shutdown(0);
+    }
+
+    public void shutdown(long awaitTerminateMillis) {
```

Comment:
@zhouxinyu  

I know and agree to your concern, actually, as I my previous comment, maybe we could add an configuration for awaitMills, what do you think ? The default value could be 0 to keep the same behavior